tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+
+envlist = py35, coverage
+
+[testenv]
+
+usedevelop = True
+
+deps =
```

Comment:
The point of moving to tox is to re-synchronize monitoring-scripts static analysis with Triscord's. Here you seem to have removed pylint and pep8.